lib/immutable-struct.rb

```diff
@@ -81,8 +81,25 @@ def self.new(*attributes,&block)
         end
       end
 
+      @@aliases = {}
```

Comment:
I've never really used a class variable in Ruby before, but I think it's appropriate here. It's roughly equivalent (in my mind) to module attributes from Elixir (for lack of a better example).

Reply:
Please double check if it's thread safe in your use case.